### DISCUSSION TITLE
Can deepgram automatically remove the words "stop" and "full stop" from the end of a sentence?

### DISCUSSION THREAD

**Discussion by Mufasa** [deepgram/community#201] [General help] [answered]
I am a newbie and am evaluating deepgram for use in a dictation transcription service that I am writing. My customers currently send their dictations to a service where these are manually transcribed and then sent back to them. They are used to saying "stop" or "full stop" at the end of their sentences.

Is there a way of configuring deepgram to automatically remove these words from the transcript?

**Comment by jkroll-deepgram** [comment]
The [Find and Replace](https://developers.deepgram.com/docs/find-and-replace) feature enables you to find one word/phrase in the transcript, and replace it with another word/phrase.

The API parameter `replace` is used, such as `replace=kpis:Key%20Performance%20Indicators`.

A curl example using this replacement (within the `url`):
```
curl \
  --request POST \
  --header 'Authorization: Token YOUR_DEEPGRAM_API_KEY' \
  --header 'content-type: application/json' \
  --data '{"url":"https://developers.deepgram.com/data/audio/interview_speech-analytics.wav"}' \
  --url 'https://api.deepgram.com/v1/listen?replace=kpis:Key%20Performance%20Indicators'
```

In your case, you can specify two replacements, such as: 
```
--url 'https://api.deepgram.com/v1/listen?replace=stop:&replace=full%20stop:'
```

**Comment by Mufasa** [comment]
@jkroll-deepgram Thank you for your suggestion. However, I need to ensure that these words are only removed when they appear at the end of a **sentence**. For example, in the sentence:
_"You will need to stop taking these pills after 3 days stop."_
It's only the second **stop** that should be removed giving us:
_"You will need to stop taking these pills after 3 days."_

In addition, I am using the javascript API and not curl. I have raised a separate issue regarding how to specify multiple words to replace in the API here: https://github.com/orgs/deepgram/discussions/203

**Reply by jkroll-deepgram** [reply, marked as answer]
@Mufasa Thanks for the clarification. You could add a leading space and trailing period into the find-term to limit this replacement to only the end of a sentence, such as `replace=%20stop.:.`, which would replace " stop." with just the period ".". However a potential edge cases could be where "stop" is the end of the meaningful sentence, like "At the red light I needed to stop." which would get replaced with "At the red light I needed to." The Find and Replace feature doesn't have the option to add any more complex rules, it just replaces each occurrence.
